### PR TITLE
Upgrade to cypress 7.0.0

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** updates cypress to 7.0.0

Addresses [CUMULUS-2282: not really](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2282)

## Changes

* Upgrade cypress version
* Update matchers for cy.intercepts according to migration notes. 

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

**Review by dopeters — APPROVED** [review]
Read the migration guide and this looks good to me. Doesn't look like there's a docker image for it yet, but we should probably upgrade to 7.0.1 when we can since it looks like there are some bug fixes: https://docs.cypress.io/guides/references/changelog#7-0-1

Excited to see what we can do with the component test runner!

**Comment by flamingbear** [comment]
@dopeters Yup, I was waiting for it, but it didn't appear until minutes after I merged this. 
https://github.com/nasa/cumulus-dashboard/pull/950

**Pull request merged**
